features/pom.xml

```diff
@@ -39,60 +39,28 @@
     </modules>
 
     <properties>
-        <!--START versions for camel features in platform-app features
-        Copied from apache-camel-2.19.4-src/parent/pom.xml
+        <!-- START versions for camel features
+        Copied from org.apache.camel:camel-parent:pom:${camel.version}
```

Comment:
👍 I reviewed the diff with https://github.com/apache/camel-karaf/blob/camel-karaf-3.4.4/pom.xml and https://github.com/apache/camel/blob/camel-3.4.4/camel-dependencies/pom.xml. There are a few versions that were bumped from what camel uses, but there are no older versions from what camel is using. I also confirmed that the remaining camel version properties are all used.